source/management.rst

```diff
@@ -20,9 +20,11 @@
 
 - PyLint Administrator(s):
 
-  * Claudiu Popa
-  * Florian Bruhin
-  * Sylvain Thénault
+  * `Claudiu Popa <https://github.com/PCManticore>`_
+  * `Florian Bruhin <https://github.com/The-Compiler>`_
+  * `Sylvain Thénault <https://github.com/sthenault>`_
+  * `Guillaume Peillex <https://github.com/hippo91>`_
+  * `Pierre Sassoulas <https://github.com/Pierre-Sassoulas>`_
```

Comment:
Why are these links when nothing else is here?

Reply:
Fixed.